guides/WRITING-DOCUMENTATION.md

```diff
@@ -77,6 +77,7 @@ To extend the default functionality of KSS, we've implemented support for custom
 | **`@responsive`**           | `@responsive`                                                | Renders breakpoint toggles for the markup example.                                                                                                     |
 | **`@status [NAME]`**        | `@status Draft`                                              | Displays a status badge. `[NAME]` must be one of the following values: `Draft`, `Work in progress`, `Ready`, `Deprecated`.                             |
 | **`@uswds [URL]`**          | `@uswds https://designsystem.digital.gov/components/button/` | `[URL]` is the link to the corresponding US Web Design System component.                                                                               |
+| **`@cmsds [URL]`**          | `@cmsds https://design.cms.gov/components/button/`           | `[URL]` is the link to the corresponding CMS Design System component.                                                                                  |
```

Comment:
If we are automatically populating this for child design systems, do we still need to add this functionality?

Reply:
No i don't think so